src/schemas/api-reference.json

```diff
@@ -1,9 +1,51 @@
 {
-    "$id": "http://schemas.taskcluster.net/base/v1/api-reference.json#",
     "$schema": "http://json-schema.org/draft-06/schema#",
     "title": "API Reference File",
     "description": "Reference of methods implemented by API",
     "type": "object",
+    "definitions": {
+      "subExpression": {"type": "array", "items": {"$ref": "#/definitions/scopeExpressionTemplate"}},
+      "scopeExpressionTemplate": {
+        "oneOf": [
+          {"type": "string", "pattern": "^[\\x20-\\x7e]*$"},
+          {
+            "type": "object",
+            "properties": {
+              "AnyOf": {"$ref": "#/definitions/subExpression"}
```

Comment:
Please copy paste: `{"type": "array", "items": {"$ref": "#/definitions/scopeExpressionTemplate"}}`

Then we don't have to discuss the name `subExpression` :)